alma_sbom.py

```diff
@@ -49,6 +32,63 @@ class PackageNevra:
     release: str = None
     arch: str = None
 
+    def __repr__(self):
+        if self.epoch is not None:
+            return f'{self.epoch}:{self.name}-' \
+                   f'{self.version}-{self.release}.{self.arch}'
+        else:
+            return f'{self.name}-{self.version}-' \
+                   f'{self.release}.{self.arch}'
+
+
+class UnsupportedFileFormatException:
```

Comment:
This is not being used. But, If you have plans to do so, I presume you want to create an exception, hence this class should inherit from `Exception` class, isn't it?

Reply:
I wanted to use them, but changed my mind and forgot to delete them =)